pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -154,15 +151,6 @@ func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
 	return err
 }
 
-func (h *NetworkUtilsHandler) IsIpv6Enabled() bool {
-	podIp := os.Getenv("MY_POD_IP")
-	if !netutils.IsIPv6String(podIp) {
-		log.Log.V(5).Info("Since the pod ip is non IPv6, IPv6 is disabled")
-		return false
-	}
-	return true
-}
-
```

Comment:
why move this function from `common` to `network` ? 

If it is really needed (not sure), I think it deserves to be done in an isolated commit, where the motivation for it is explained.

Reply:
its needed in order use IsIpv6Enabled in several places - but have one function
it was groomed with Alona , and i need it in this PR please

Reply:
> its needed in order use IsIpv6Enabled in several places - but have one function
> it was groomed with Alona , and i need it in this PR please
> since its an integral part
> h isnt used and not needed (was groomed with Petr that its logical to create a standalone function), also the mock changes arent needed (once removed it from here, the mock generate removed them from the autogen files)

OK, makes sense - the function doesn't require any real HW behind, and as such doesn't need to be on the handler. I'm OK with it.

But anyway you look at this, it deserves to be on a separate commit, with that reasoning written in the commit msg. That commit would, of course, be part of this PR.

Reply:
i think otherwise, and also as part of this task wont fix sriov migration which wasnted tested, as i understood, 
i did the changes i agreed with, every other change must be accepted by other reviewers, and be coordinated with Petr / Alona if they want me to extend the task scope
thanks